packages/cli/src/runIOS/findMatchingSimulator.js

```diff
@@ -39,16 +39,24 @@ function findMatchingSimulator(simulators, simulatorString) {
   }
 
   let match;
-  for (const version in devices) {
+  for (const versionDescriptor in devices) {
+    const device = devices[versionDescriptor];
+    let version = versionDescriptor;
+
+    if ((/^com\.apple\.CoreSimulator\.SimRuntime\./g).test(version)) {
```

Comment:
Is there a chance this namespace can be different in some environments?

Reply:
Thanks for detailed writeup! I tried digging around myself too, but was unable o find any official notes from Apple about this braking change. Again, I was just curious to understand the rationale behind this change, to make sure we adopt the CLI code to function appropriately. 

In this case, after your explanation, I am fine with supporting the current use case and extending the regular expression when there's such need.